scss/reset/_forms.scss

```diff
@@ -81,15 +81,6 @@ label {
 	}
 }
 
-.label--field {
```

Comment:
Yes, but these same styles are in `assets/scss/components/_forms.scss` in `meetup-web-components`